### ISSUE TITLE
Add TypeScript support

### ISSUE THREAD

**Issue by opensas** [sveltejs/svelte.dev#853]
Now that official TypeScript support for Svelte has been announced, It would be very convenient to also add it to the Repl.

The Repl has become a very useful tool, not only to showcase applications and share code snippets, but also to report issues.

**Comment by opensas** [comment]
Great to know, it really is a very useful tool.

**Comment by bradphelan** [comment]
This would be a great addition

**Comment by samuelandert** [comment]
Indeed, this is one of the most interesting and anticipated REPL features.

**Comment by danawoodman** [comment]
Will this be considered by the core team? Would be pretty rad 😸

**Comment by flxmrk** [comment]
Wold be nice for sure!  Would make REPL useful again!

**Comment by mawoka-myblock** [comment]
> Wold be nice for sure! Would make REPL useful again!

It is definitely not useless, but TypeScript support would be great!

**Comment by ehartford** [comment]
I will send you a gift basket

**Comment by gyurielf** [comment]
Any update on this ?
It's still awaited feature.

**Comment by braebo** [comment]
Is this planned?

**Comment by teemingc** [comment]
closed by https://github.com/sveltejs/svelte.dev/pull/1277
